source/fundamentals/crud/compound-operations.txt

```diff
@@ -48,10 +48,14 @@ configurable :ref:`sort <node-fundamentals-sort>` and
 
    Starting in version 5.7, you can set the ``includeResultMetadata``
    setting in the ``options`` object to specify the return type for each
-   of these methods. The setting defaults to ``true``, which means the
-   method returns a ``ModifyResult`` type. If you set
-   ``includeResultMetadata`` to ``false``, the method returns the
-   modified document.
+   of these methods.
+   
+   Starting in version 6.0, the setting defaults to
+   ``false``, which means each method returns either the matched document
+   or ``null`` if no document is matched. If you set
```

Comment:
S: To reduce ambiguity. Definitely open to alternatives!

```suggestion
   ``false``, which means each method returns the matched document,
   if one is found, or ``null``. If you set
```

Reply:
edited slightly

Reply:
not a blocking change, but there's still ambiguity around what the method returns and under which conditions. i.e., this sentence could also mean "If no document is matched, the method returns either the matched document or null." 

putting the condition in a parenthetical tied to the first case (what i suggested above) is one way to handle this, but there are probably others.

Reply:
Oddly, "parenthetical" can mean any [interjection of qualifying or explanatory information](https://www.collinsdictionary.com/us/dictionary/english/parenthetical)--even when it's offset by commas. That's what I meant here.